lib/node/NodeSourcePlugin.js

```diff
@@ -35,54 +33,60 @@ module.exports = class NodeSourcePlugin {
 				return ParserHelpers.addParsedVariableToModule(this, name, mockModule + suffix);
 			});
 		}
+		try {
```

Comment:
Don't wrap this is a try-catch block.

Reply:
I'd need the try catch for the require, but I can see it making sense to limit it as much as possible. If we really want to leave nodeLibsBrowser as a constant then the only option I can think of would be to wrap the require in a function that has the try/catch and returns null if it gets an exception. What do you think of that?